src/vertex.cpp

```diff
@@ -12,20 +9,20 @@ Vertex::Vertex(const unsigned int &id, Coordinates c) : id(id), dist(constants::
  * @param length - Edge length
  * @return Pointer to the new Edge created
  */
-std::shared_ptr<Edge> Vertex::addEdge(const std::shared_ptr<Vertex> &d, double length) {
+/*std::shared_ptr<Edge> Vertex::addEdge(const std::shared_ptr<Vertex> &d, double length) {
     auto newEdge = std::make_shared<Edge>(std::make_shared<Vertex>(*this), d, length);
     adj.push_back(newEdge);
-    d->incoming.push_back(newEdge);
     d->indegree++;
     return newEdge;
-}
+}*/
 
 /**
  * Removes an outgoing edge, with a given destination, from the Vertex
  * Time Complexity: O(indegree(v) * outdegree(v))
  * @param destID - Id of the destination Vertex of the Edge to be removed
  * @return True if successful, and false if no such Edge exists
  */
+/*
```

Comment:
Shouldn't we just remove everything related to Edge objects? Instead of commenting, its simpler just removing

Reply:
Yes, I left it in case you guys suggested changes and I needed to go back